### PR TITLE
Add network interface monitoring integration

### PR DESCRIPTION
**Pull request by calebhailey**

### CODE REVIEW

File: integrations/system/network-interface-monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+<!-- Please provide links to any relevant reference documentation to help users learn more and/or troubleshoot this integration; specifically including any third-party software documentation. -->
+
+1. This integration uses [Sensu Tokens][tokens] for variable substitution.
+
+<!-- Links -->
+[check]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/checks/
+[asset]: https://docs.sensu.io/sensu-go/latest/plugins/assets/
+[subscription]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/subscriptions/
+[subscriptions]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/subscriptions/
```
Comment (jspaleta) [inline integrations/system/network-interface-monitoring/README.md:163]: Reference Thunderdome!!!!!!! 
subscription  refer not used... duplicate

File: integrations/system/network-interface-monitoring/sensu-resources.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+metadata:
+  name: network-interface-health
+spec:
+  command: >-
+    network-interface-checks
+    --state-file /var/cache/sensu/sensu-agent/network-interface-checks
+  runtime_assets:
+  - sensu/network-interface-checks:0.1.0
+  publish: true
+  subscriptions:
```
Comment (jspaleta) [inline integrations/system/network-interface-monitoring/sensu-resources.yaml:13]: Do the subscriptions match the README?

File: integrations/system/network-interface-monitoring/README.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+This integration is compatible with the [Sumo Logic Host monitoring dashboard][sumo-host-dashboard-link] (included w/ [Sensu Plus][sensu-plus]).
+
+![](img/dashboard.png)
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Add one of the following [subscriptions] to [agents] that should run this check.
```
Comment (jspaleta) [inline integrations/system/network-interface-monitoring/README.md:27]: Do the README subscriptions match the CheckConfig?

File: integrations/system/network-interface-monitoring/README.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+<!-- Sensu Integration description; supports markdown -->
+
+The network interface monitoring integration provides metrics collection and alerting for network interface activity.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `network-interface-health` ([check])
+* `sensu/network-interface-check` ([asset])
```
Comment (jspaleta) [inline integrations/system/network-interface-monitoring/README.md:12]: Should this be versioned?

**Review by jspaleta — CHANGES_REQUESTED** [review]

**Comment by jspaleta** [comment]
Add an initial CHANGELOG

File: integrations/system/network-interface-monitoring/sensu-resources.yaml (hunk 2 above)
Comment (calebhailey) [reply integrations/system/network-interface-monitoring/sensu-resources.yaml:13]: Good catch; need to add `system`.

File: integrations/system/network-interface-monitoring/README.md (hunk 3 above)
Comment (calebhailey) [reply integrations/system/network-interface-monitoring/README.md:27]: Oops, good catch.

**Review by jspaleta — APPROVED** [review]

**Pull request merged**